src/code/Utils.cs

```diff
@@ -21,6 +22,13 @@ internal static class Utils
     {
         #region Members
 
+        private const string NoVaultRegistered = @"
+        There are currently no extension vaults registered.
+        At least one vault must be registered before SecretManagement can add or retrieve secrets.
+        You can download SecretManagement extension vault modules from PowerShellGallery.
+        https://aka.ms/SMVaults
```

Comment:
I would prefer we spelled out: https://aka.ms/SecretManagementVaults since folks are likely to cut and paste it anyways.